### PR TITLE
[Quest]RDM AF3 Peace Spirit Conv

### PR DESCRIPTION
**Pull request by hooksta4**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Converts RDM AF3 quest to IF.
- Removes all NPC lua dialog associated with quest
- Adds defaultActions for those NPC's
- Reworked dialogs for Guardian Statue based on [capture](https://www.youtube.com/watch?v=u_p6Vv_pDH0) 
    - Players can pop NM, zone and repop NM allowing for others to get the Nail Puller.  Until the player on quest trades the Nail Puller to the box, they can do this indefinitely. https://ffxiclopedia.fandom.com/wiki/Talk:Peace_for_the_Spirit
    - TODO Verify If you get the message "Something feels wrong, but nothing happens.", click off Sneak.
    - This message shows if clicking on the box if the NM is alive.
    - Set localVar to only allow player to pop once per zone. 

## Steps to test these changes
Complete AF2 quest and speak with Curilla. 
Follow questline.

### CODE REVIEW

**Comment by Frankie-hz** [comment]
All AF quest conversions have been working on HXI since implementation with no issues/reports.

File: scripts/quests/sandoria/RDM_AF3_Peace_for_the_Spirit.lua [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+-----------------------------------
+-- Log ID: 0, Quest ID: 86
+-- Curilla !pos 27 0.1 0.1 233
+-- Sharzalion !pos 95 0 111 230
+-- Daggo !pos 89 1 119 230
+-- Miser Murphy !pos -14 -16 70 204
+-- Dry Fountain !pos -14 -16 70 204
+-- Oaken Box !pos -164 0 224 200
+-----------------------------------
+local ID = zones[xi.zone.FEIYIN]
```
Comment (Frankie-hz) [inline scripts/quests/sandoria/RDM_AF3_Peace_for_the_Spirit.lua]: Can you make this `feiyinID` to keep consistency throughout the files.

Comment (hooksta4) [reply scripts/quests/sandoria/RDM_AF3_Peace_for_the_Spirit.lua]: That is a great catch, Frank. I have updated to ensure consistency. For others, I will update as well in the future. Thanks!

**Comment by hooksta4** [comment]
Can we re run the Full startup check? Is there anything else I can do to get the Big Red X removed so someone reviews it? I added something and it ran fine this time ;)

**Pull request closed without merging**